components/ui/components/floating/UserCard.tsx

```diff
@@ -1,24 +1,96 @@
 import { useFloating } from "solid-floating-ui";
 import { For, JSX, Ref, Show, createSignal } from "solid-js";
 import { Portal } from "solid-js/web";
-import { styled } from "solid-styled-components";
-
+import { ThemeProvider, styled } from "solid-styled-components";
 import { autoUpdate, flip, offset, shift } from "@floating-ui/dom";
 import { Motion, Presence } from "@motionone/solid";
 import { ServerMember, User } from "revolt.js";
-
 import { ColouredText } from "../design";
 
 /**
  * Base element for the card
  */
 const Base = styled("div", "Tooltip")`
   color: white;
-  background: black;
   width: 400px;
   height: 400px;
+  background: ${({ theme }) => theme!.colours["background-300"]};
+`;
+
+const Container = styled.div`
+  position: relative;
+  width: 24px;
+  height: 24px;
+`;
+
+const Dots = styled.svg`
+  fill: white;
+  width: 100%;
+  height: 100%;
+`;
+
+const Dot = styled.circle`
+  fill: currentColor;
+`;
+
+
+const profiletopcontainerStyle = {
+  position: 'relative',
+  width: '100%',
+  height: '350px',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
+  marginBottom: '20px'
+};
+
+function getStatusColor(status: string | null | undefined) {
```

Comment:
Shouldn't the colors here be changed to the ones from the current theme?

Reply:
Was namely just place holding, I've lost intrest for now.